app/src/main/java/org/oppia/app/fragment/FragmentComponent.kt

```diff
@@ -24,4 +25,5 @@ interface FragmentComponent {
   fun inject(homeFragment: HomeFragment)
   fun inject(stateFragment: StateFragment)
   fun inject(conceptCardFragment: ConceptCardFragment)
+  fun inject(conceptCardFragment: ContentListFragment)
```

Comment:
Check this variable name, it should be `contentListFragment`

Reply:
corrected variable name